cmd/dump.go

```diff
@@ -25,10 +26,21 @@ import (
 	"github.com/urfave/cli"
 )
 
-func addFile(w archiver.Writer, filePath, absPath string, verbose bool) error {
+func addReader(w archiver.Writer, r io.ReadCloser, info os.FileInfo, customName string, verbose bool) error {
 	if verbose {
-		log.Info("Adding file %s\n", filePath)
+		log.Info("Adding file %s\n", customName)
```

Comment:
```suggestion
		log.Info("Adding file %s", customName)
```

although it came from old code, the `\n` seems incorrect.